README.md

```diff
@@ -111,10 +109,10 @@ and run Elasticsearch-related tests against your own server using the
 
     > mvn clean install -Dtest.elasticsearch.host.provided=true -Dtest.elasticsearch.host.url=http://localhost:9200
 
-If you want to run tests against an older Elasticsearch version  (2.x for instance),
+If you want to run tests against a different Elasticsearch version  (6.x for instance),
 you will still have to select a profile among those listed above, and disable the default profile:
 
-    > mvn clean install -P!elasticsearch-5.2,elasticsearch-2.2 -Dtest.elasticsearch.host.provided=true -Dtest.elasticsearch.host.url=http://localhost:9200
+    > mvn clean install -P!elasticsearch-5.6,elasticsearch-6.0 -Dtest.elasticsearch.host.provided=true -Dtest.elasticsearch.host.url=http://localhost:9200
```

Comment:
Here you negate the `elasticsearch-5.6` profile, but you don't do it there: https://github.com/hibernate/hibernate-search/pull/1749/commits/6e9a18363128d6eefd754277a705033f4a5fe677#diff-600376dffeb79835ede4a0b285078036R2003 .

Reply:
Right, that's because it's not necessary when the version is provided. I'll add it everywhere for consistency.

Reply:
On second thought I won't, it just adds one more string that we may forget to update...